src/adaptors/throttle.rs

```diff
@@ -86,12 +86,14 @@ const DELAY: Duration = Duration::from_millis(250);
 /// [@BotSupport]: https://t.me/botsupport
 #[derive(Copy, Clone, Eq, PartialEq, Hash)]
 pub struct Limits {
-    /// Allowed messages in one chat per second
-    pub chat_s: u32,
-    /// Allowed messages per second
-    pub overall_s: u32,
-    /// Allowed messages in one chat per minute
-    pub chat_m: u32,
+    /// Allowed messages in one chat per second.
+    pub messages_per_sec_chat: u32,
+
+    /// Allowed messages in one chat per minute.
+    pub messages_per_min_chat: u32,
+
+    /// Allowed messages per second.
+    pub messages_per_sec_overall: u32,
```

Comment:
What's the point in blank lines? Haven't seen blank lines in structures in our codebase before :thinking:

Reply:
With blank lines, the fields are easier to separate from each other. Compare with this:

```rust
/// Allowed messages in one chat per second.
pub messages_per_sec_chat: u32,
/// Allowed messages in one chat per minute.
pub messages_per_min_chat: u32,
/// Allowed messages per second.
pub messages_per_sec_overall: u32,
```

We already have blank lines in all the API types.

Reply:
Nvm then, I'm blind